Changelog.md

```diff
@@ -14,6 +14,8 @@ Compiler Features:
  * Static Analyzer: Do not warn about unused variables or state mutability for functions with an empty body.
  * Type Checker: Add an additional reason to be displayed when type conversion fails.
  * Yul: Support object access via ``datasize``, ``dataoffset`` and ``datacopy`` in standalone assembly mode.
+ * Commandline interface: Adds new option ``--new-reporter`` for improved diagnostics formatting
```

Comment:
Needs to be moved.

Reply:
I assume you mean it should be alphabetically ordered. changed that. (but other entries seem to conflict, too, I realized)